lib/cdo/shared_constants/pd/facilitator1819_application_constants.rb

```diff
@@ -48,16 +48,16 @@ def self.clean_multiline(string)
          For more details about the requirements to facilitate each program, please visit the
          [2018-19 Facilitator Development Program Description](https://docs.google.com/document/d/1aX-KH-t6tgjGk2WyvJ7ik7alH4kFTlZ0s1DsrCRBq6U).'
       ),
-      planOnTeaching: 'Do you plan on teaching this course in the 2018-19 school year?',
-      abilityToMeetRequirements: clean_multiline(
+      plan_on_teaching: 'Do you plan on teaching this course in the 2018-19 school year?',
```

Comment:
This _shouldn't_ break the question numbering, but it's worth checking on your machine to make sure. Please remerge staging to verify